flashinfer/mla/_sparse_mla_sm120.py

```
@@ -0,0 +1,905 @@
+    try:
+        mtime = path.stat().st_mtime
+    except OSError:
+        return
+    if mtime <= _decode_dsv4_cache_mtime:
+        return
+    try:
+        AutoTuner.get().load_configs(str(path))
+        _decode_dsv4_cache_mtime = mtime
+    except Exception:
+        # Keep mtime unchanged so the next cold call retries.
+        pass
```

Ruff (0.15.13)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)